pkgs/applications/editors/vim/plugins/nvim-treesitter/generated.nix

```diff
@@ -2121,12 +2121,12 @@
   };
   sql = buildGrammar {
     language = "sql";
-    version = "0.0.0+rev=5f928f4";
+    version = "0.0.0+rev=9fe5aea";
     src = fetchFromGitHub {
       owner = "derekstride";
       repo = "tree-sitter-sql";
-      rev = "5f928f404d2aa024abce8657778fc10c03f1511f";
-      hash = "sha256-7W6vuaZjDZgoaxJexPPBjJZlutlTT+hTFL1dq9k2NSo=";
+      rev = "9fe5aeaa8d58d00cc31c20a3ae923ae695ce2ce7";
```

Comment:
can you please update again to include d8fffdf0902bf55994fd2e8a0a46e221ca988589 thanks !

Reply:
Should be good now.

Reply:
still pointing to 9fe5
latest is d8fffdf

```
commit d8fffdf0902bf55994fd2e8a0a46e221ca988589 (origin/gh-pages)
Author: matthias-Q <matthias-Q@users.noreply.github.com>
Date:   Wed Nov 15 13:59:32 2023 +0000

    deploy: d234688b2414990dbd97d4a046331f37af111666

commit 9fe5aeaa8d58d00cc31c20a3ae923ae695ce2ce7
Author: DerekStride <DerekStride@users.noreply.github.com>
Date:   Mon Nov 13 21:03:28 2023 +0000

    deploy: b7148b32c2fb87e1cf46baad439e567586464b5a
```

Reply:
Hmm weird, because I just ran the script...

Reply:
don't worry much about it, its not urgent, when I get some time, I can have a look at updating it, 

shouldn't block your PR

Reply:
> Hmm weird, because I just ran the script...

That might be because of the caching the update script does